### ISSUE TITLE
Dns tests fail on build behind a proxy

### ISSUE THREAD

**Issue by mkunz7** [sandia-minimega/minimega#669]
```
TESTING
?   	apigen	[no test files]
?   	bridge	[no test files]
--- FAIL: TestTCPRtt (2.00s)
	issue_test.go:17: read udp 10.0.8.27:49394->8.8.4.4:53: i/o timeout
--- FAIL: TestClientRemote (2.00s)
	remote_test.go:14: failed to exchange: read udp 10.0.8.27:42401->176.58.119.54:53: i/o timeout
FAIL
```

**Comment by mkunz7** [comment]
176.58.119.54 -> deb.atoom.net???

**Comment by jcrussell** [comment]
This is "fixed" by #672 -- `test.bash` won't run tests on vendorized code anymore.
